src/infrastructure/commonDiConfig.ts

```diff
@@ -21,7 +21,8 @@ import type { Config } from './config'
 import type { ExternalDependencies } from './diConfig'
 import { SINGLETON_CONFIG } from './diConfig'
 
-export type CommonDiConfig = Record<keyof CommonDependencies, Resolver<unknown>>
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type CommonDiConfig = Record<keyof CommonDependencies, Resolver<any>>
```

Comment:
What's the reason for this?

Reply:
https://github.com/kibertoad/awilix-manager/commit/5ca93b6a6fae2d4fc02ff3fd4d22149980490906 this change unfortunately threw off the TS type inference :-/

Reply:
yikes but alright